packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -75,13 +104,32 @@ test.before(async () => {
         .then((record) => indexer.indexPdr(esClient, record, esAlias))
     )
   );
+
+  // Create a PG Collection
+  t.context.collectionCumulusId = 1234;
+  t.context.testPgCollection = fakeCollectionRecordFactory({
+    cumulus_id: t.context.collectionCumulusId,
+  });
```

Comment:
I think this is what I had in my test-granules.js. Mark had a good point that this is kind of an anti-pattern. If we get into this habit of setting `cumulus_id` manually we could theoretically run into conflicts if we create a test later that also happens to generate the same cumulus_id. This might be cleaner:

```
  // Create a PG Collection
  t.context.testPgCollection = fakeCollectionRecordFactory();
  const collectionPgModel = new CollectionPgModel();
  [t.context.collectionCumulusId] = await collectionPgModel.create(
    t.context.knex,
    t.context.testPgCollection
  );
```